tensorflow/java/src/main/java/org/tensorflow/types/TFBool.java

```diff
@@ -0,0 +1,14 @@
+// GENERATED FILE. Edit tftypes.pl instead.
+package org.tensorflow.types;
+
+/** The class TFBool represents Boolean. */
+public class TFBool implements Types.TFType {
+  /** Represents the type TFBool at run time. */
+  public static final Class<TFBool> T = TFBool.class;
```

Comment:
What kinds of such uses do you foresee? My inclination is to cut down on layers of abstraction where the value isn't clear. In this case, my understanding is that `TFBool.T` is meant to be fed into methods that require a `Class<TFBool>` parameter. Wouldn't that imply that the class object must be the representative of the type? 

Or am I missing something?